chain-impl-mockchain/src/accounting/account/mod.rs

```diff
@@ -263,13 +273,27 @@ mod tests {
                 .take(stake_pool_size)
                 .collect::<Vec<_>>();
 
+            let voting_tokens_size = usize::arbitrary(gen);
+            let arbitrary_voting_tokens = iter::from_fn(|| Some(TokenIdentifier::arbitrary(gen)))
```

Comment:
I guess I should put a max to it, and actually thought about it, but I wasn't sure what to use for it. The accounts are also generated in the same way, though, `let account_size = std::cmp::max(usize::arbitrary(gen), 1);`, and that's been there since 2019, so I think it's probably not that risky.

Reply:
I mean, it's not too much of a problem now, the worst that can happen is that at some point it will cause a problem and we already know what to do